src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/ProvideSemanticTokensDeltaParams.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models.Proposals;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Models
+{
+    [Obsolete("Proposed for the next version of the language server. May not work with all clients.  May be removed or changed in the future.")]
```

Comment:
Is there already a work item tracking the addition of this?

Reply:
Good call, I filed https://github.com/dotnet/aspnetcore/issues/34496 to track that. I believe O# already updated `SemanticTokensDeltaParams` to remove the proposal, but taking a new O# version is a whole thing.